modules/gapi/include/opencv2/gapi/s11n/base.hpp

```diff
@@ -18,25 +18,46 @@ struct IIStream;
 
 namespace detail {
 
+//! @addtogroup gapi_serialization
+//! @{
+
 struct NotImplemented {
 };
 
-// The default S11N for custom types is NotImplemented
-// Don't! sublass from NotImplemented if you actually implement S11N.
+/** @brief This structure allows to implement serialization routines for custom types.
+ *
```

Comment:
Example code snippet would be a killer note here!

Reply:
Also, here I'd mention we have a dozen of exported `<<` and `>>` operators for the standard and G-API types which can be found in the appropriate section

Reply:
(so that the new user code could use those)

Reply:
Done